src/SfxWeb/src/app/Models/DataModels/collections/RepairTaskCollection.ts

```diff
@@ -62,7 +61,7 @@ export class RepairTaskCollection extends DataModelCollectionBase<RepairTask> {
         this.longRunningApprovalJob = longRunningApprovalRepairTask;
         this.longestExecutingJob = longRunningExecutingRepairTask;
```

Comment:
Are we using this.longestExecutingJob anywhere ? We probably need a banner for this aswell as now we usually see ICMs for this case. @saketharsh is working on a proposal to cover all these cases. Saket/Jeffrey let's discuss on the next set of updates once this proposal is ready

Reply:
@anantshankar17 we use the longest executing for one of the tiles on the repair jobs page, we could look at adding a banner for it aswell.